src/popup/router/components/InviteItem.vue

```diff
@@ -113,49 +135,65 @@ export default {
 .invite-row {
   padding: 1rem;
   width: 100%;
-  border-bottom: 1px solid #12121b;
   text-align: left;
-  color: #bcbcc4;
-  background: #21222a;
+  color: $text-color;
   position: relative;
+  border-bottom: 2px solid $black-1;
 
   .invite-link {
     margin-bottom: 5px;
-    font-size: 12px;
-    color: $text-color;
+    font-size: 11px;
+    display: flex;
 
     span {
-      width: 260px;
-      display: inline-block;
+      margin-left: 5px;
       text-overflow: ellipsis;
       overflow: hidden;
       white-space: nowrap;
+      color: $white-1;
     }
   }
 
   .invite-link-copy {
     padding: 0;
+    color: $gray-2;
   }
 
   .invite-info {
-    font-size: 14px;
-    font-weight: bold;
-    color: $white-color;
+    font-size: 13px;
     display: flex;
     align-items: center;
     margin-bottom: 10px;
+    color: $gray-2;
 
-    .primary {
-      color: $secondary-color;
+    .balance-display {
+      color: $white-1;
       margin-right: 5px;
+
+      span {
+        color: $secondary-color;
+      }
     }
 
     .date {
-      font-size: 12px;
-      font-weight: normal;
+      font-size: 11px;
       margin-left: auto;
       color: $text-color;
     }
   }
+
+  .amount-send-container {
+    margin: 0;
+  }
+
+  .centered-buttons {
+    text-align: center;
```

Comment:
I think flex suites more here because buttons are more blocks than text, and it would be looking better on different device sizes.